### DISCUSSION TITLE
is there a way to publish Jupyter notebook using vitepress

### DISCUSSION THREAD

**Discussion by fjun99** [vuejs/vitepress#3456] [Q&A]
is there a way to publish Jupyter notebook directly using vitepress?

thanks.

**Comment by brc-dd** [comment]
You can probably use nbconvert and embed the output using iframe or maybe just store them in public dir and serve those. Another alternative can be to setup jupyterlite and embed notebooks using that.

**Comment by fjun99** [comment]
> You can probably use nbconvert and embed the output using iframe or maybe just store them in public dir and serve those. Another alternative can be to setup jupyterlite and embed notebooks using that.

Thank you!

Will try nbconvert and jupyterlite。
